scylla/src/statement/prepared_statement.rs

```diff
@@ -270,6 +270,27 @@ impl PreparedStatement {
         self.config.tracing
     }
 
+    /// Make use of cached metadata to decode results
+    /// of the statement's execution.
+    ///
+    /// If true, the driver will request the server not to
+    /// attach the result metadata in response to the statement execution.
+    ///
+    /// The driver will cache the result metadata received from the server
+    /// after statement preparation and will use it
+    /// to deserialize the results of statement execution.
+    ///
+    /// This option is false by default.
+    pub fn set_use_cached_result_metadata(&mut self, should_skip: bool) {
+        self.config.skip_result_metadata = should_skip;
+    }
```

Comment:
Please change the name of the parameter.

![image](https://github.com/scylladb/scylla-rust-driver/assets/4950936/06202ba1-f11e-42c5-8c40-060efc367c37)

The name of the parameter is shown in the documentation, and it looks confusing in relation to the method's name.